samples/meetings-attendance-report/csharp/MeetingAttendance/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "MicrosoftAppId": "<<Microsoft-App-Id>>",
+  "MicrosoftAppPassword": "<<Microsoft-App-Password>>",
+  "MicrosoftAppTenantId": "<<Microsoft-App-Tenant-Id>>",
+  "UserId": "<<Granting-Policy-User-Id>>"
```

Comment:
How are we getting this user id and showing?
Mention the step in Readme file.

Reply:
It is mentioned in Readme already.